src/constants/android-settings.js

```diff
@@ -51,6 +59,7 @@ export const androidSettingsDefault = {
 }
 
 export const RESERVED_VALUES = 'reservedValues'
+export const manual = 'manual'
```

Comment:
It'd be more consistent to uppercase this variable name. Also you probably want to refer to this constant instead of using a string literal in lines 13 and 40 above.

Reply:
(this would also mean moving all the constant declarations up in the file, which would make sense to me as well)

Reply:
Thanks, Hendrik! I will fix it :)